src/index.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    "Roboto",
+    "Oxygen",
+    "Ubuntu",
+    "Cantarell",
+    "Fira Sans",
+    "Droid Sans",
+    "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: #ec4b8a;
```

Comment:
- [ ] Kindly ensure that you use the same typography as provided by the Behance design for all of your components. I've added an image below for your ease.

![image](https://user-images.githubusercontent.com/90556221/187902504-b912ded4-31ad-4c2f-87a5-d621ab367be5.png)

Reply:
It has been modified